README.md

```diff
@@ -187,7 +187,7 @@ docker run -d -p 2801:2801 -e SLACK_WEBHOOKURL=XXXX -e DATADOG_APIKEY=XXXX falco
 
 * Download the latest release:
   ```shell
-  wget https://github.com/falcosecurity/falcosidekick/releases/latest -o falcosidekick
+  wget https://github.com/falcosecurity/falcosidekick/releases/latest -O falcosidekick
```

Comment:
thanks for updating this
As you are touching this, the command is not quite correct, as this download only the release notes itself and not the binary

we have a few binaries for different architectures and also that will not work because will need to set the version as we have multiple binaries.

so you first need to get the latest release version and then use that as input for the wget to fetch the file.

and there are a few more steps that you need to do after you download, that is a .tar.gz file, you will need to decompress before doing the chmod and copy to the directory.

to fetch the latest release version we can do something simiar

```
VER=$(curl --silent -qI https://github.com/falcosecurity/falcosidekick/releases/latest | awk -F '/' '/^location/ {print  substr($NF, 1, length($NF)-1)}');
```

then use that for the get

```
wget https://github.com/falcosecurity/falcosidekick/releases/download/$VER/falcosidekick_$VER_linux_arm64.tar.gz
```

Reply:
ver=$(curl --silent -qI https://github.com/falcosecurity/falcosidekick/releases/latest | grep location: | sed 's|.*/\(.*\)|\1|')

Reply:
VER=$(curl --silent -qI https://github.com/falcosecurity/falcosidekick/releases/latest | awk '/^location/ {match($0, /\/([^\/]+)$/, arr); print arr[1]}')

But failing putting that into wget still

Reply:
need to make that work :)

Reply:
```
VER=$(curl --silent -qI https://github.com/falcosecurity/falcosidekick/releases/latest | awk -F '/' '/^location/ {print  substr($NF, 1, length($NF)-1)}')

wget https://github.com/falcosecurity/falcosidekick/releases/download/$VER/falcosidekick_${VER}_linux_arm64.tar.gz

or

wget https://github.com/falcosecurity/falcosidekick/releases/download/$VER/falcosidekick_${VER}_linux_amd64.tar.gz
```

With these syntaxes, it works.